examples/qwik/src/root.tsx

```diff
@@ -0,0 +1,29 @@
+import { component$, useStyles$ } from '@builder.io/qwik';
+import { QwikCityProvider, RouterOutlet, ServiceWorkerRegister } from '@builder.io/qwik-city';
+import { RouterHead } from './components/router-head/router-head';
+
+import globalStyles from './global.css?inline';
```

Comment:
@hhthacker we can for the purpose of this example remove the styles and follow the semantics of the react-apollo-live examples.

Later this month we should create a shared stylesheet across examples so they follow the same theme and link to other examples. Will need @designhorf to pitch in.

Reply:
removed styles!